test/integration/context_managers_tests.py

```diff
@@ -31,10 +35,16 @@ def test_context_manager_for_statements(connection):
 
 @pytest.mark.exceptions
 @pytest.mark.context_managers
-def test_context_manger_closes_everything_on_exception(dsn, user, password, schema):
+def test_context_manger_closes_everything_on_exception(
+    dsn, user, password, schema, websocket_sslopt
+):
     with pytest.raises(ExaQueryError):
         with pyexasol.connect(
```

Comment:
Maybe for these cases we should introduce a fixture that returns connection factory